feature/ec2/imds/api_client.go

```diff
@@ -1,4 +1,4 @@
-package ec2imds
```

Comment:
is this change required? Kinda like `ec2imds` parity with v1 sdk.

Reply:
good point I'll look at the doc branch and update these.

Reply:
Wait, was your ^ comment for documentation update ask,  OR on my comment to not change package name from `ec2imds` to `imds`?

Reply:
https://github.com/aws/aws-sdk-go-v2/pull/986